### PR TITLE
{chem}[foss/2017b] NAMD v2.12

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/35b816e25cd0399beef4501e78b9f56d for a full test report.

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**FAILED**
Build succeeded for 0 out of 7 (1 easyconfigs in this PR)
lg-1r17-n06 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz, Python 2.6.6
See https://gist.github.com/73617f287be1ccb8b1d9f08aad2c1e97 for a full test report.

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**FAILED**
Build succeeded for 6 out of 7 (1 easyconfigs in this PR)
lg-1r17-n06 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz, Python 2.6.6
See https://gist.github.com/ef9d1bd0f595d85a81235684a789d592 for a full test report.

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
lg-1r17-n06 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz, Python 2.6.6
See https://gist.github.com/5d791329e54bcba1ab63ada073344175 for a full test report.

File: easybuild/easyconfigs/n/NAMD/NAMD-2.12-foss-2017b-mpi.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+sources = [{
+    'filename': 'NAMD_%(version)s_Source.tar.gz',
+    'extract_cmd': "tar xfv %s",  # source file is actually not gzipped
+}]
+checksums = ['3be34f04c64a1e4225c04560c676ca8ff9bb57aa8830e42a46bf9ac6b16eb403']
+
+dependencies = [
+    ('Tcl', '8.6.7'),
+    ('FFTW', '3.3.6'),
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/n/NAMD/NAMD-2.12-foss-2017b-mpi.eb]: This is inherited from the existing `NAMD-2.12-foss-2017a-mpi.eb`, but I doubt there's a good reason for it (can't think of something that would require this).
It's a bit silly to do this, but it doesn't hurt, so not worth blocking the PR over it imho...

@vanzod If you want to clean this up, please do so consistently in a follow-up PR.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2064.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1dc3fde3ed6e0611e20f6519d17ff802 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2699.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e7ae47d6e443c90c6b19ddcf723979f8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
node3117.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/3fa3c82eecb6dc231bb314be08e2263c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**